### PR TITLE
Here is a pretty signup page

### LINKED ISSUE #212: Create a sensible layout with correct text for the signup pages
Currently, it looks very ugly

### PR DESCRIPTION
**Pull request by feeds**
Go and check it out, fixes #212

### CODE REVIEW

**Comment by feeds** [comment]
Known bug: JS does not fade out if the checkbox is not selected.
@maltezacharias How can one fix this?
-- fixed

**Comment by feeds** [comment]
Major feedback on the layout can be submitted, but small tweaks are not for now ;)

**Comment by bjrne** [comment]
I know you said small tweaks are not for now, but maybe they are fixed in a minute, otherwise just consider this a a note to myself ;)

This looks weird when loading the page
Screenshot from 2020-08-13 09-07-00
Its layout however works when unchecked:
Screenshot from 2020-08-13 09-08-23

Minor thing, this could use a bit of spacing:
Screenshot from 2020-08-13 09-08-59

**Review by bjrne — APPROVED** [review]
Fine so far, worlds better than before.

However, can you explain why "Time Availability Per Week" has "----------" preselected and the Medstud Experience Level gives "Preclinical Section" althought "-------" appears as the first item in the dropdown when clicked?

**Comment by bjrne** [comment]
Notes for you: 
- [x] Cache layout
- [x] Fix translations
- [x] Fix tests
- [x] Spacing under email
- [x] Spacing between property groups
- [x] breaking glitch in MedStuden
- [x] Preselected "Clinical Section"

File: backend/apps/matching/configuration/properties.py [hunk 1]
```diff
@@ -163,6 +200,36 @@ def generate_random_assignment(self, rs=None):
             *chain(*[p.generate_random_assignment(rs) for p in self.properties]),
         ]
 
+    def _get_signup_layout(self, prefix=None):
+        field_names = self.get_model_field_names(prefix)
+        conditional_field = field_names[0]
+        print(field_names)
```
Comment (bjrne) [inline backend/apps/matching/configuration/properties.py]: :imp:

**Comment by feeds** [comment]
The preselection is actually intendend
`
m4e.OrderedSingleChoiceProperty(
                            name="experience",
                            label="Experience Level",
                            # no_choice_option=True,
                            # no_choice_label="None chosen",
                            choices=[
                                (0, "Preclinical Section"),
                                (1, "Last Year Student"),
                                (2, "Assistant Doctor"),
                                (3, "Consultant"),
                            ],
                            default=0,
                        ),
`

**Pull request merged**